proto/README.md

```diff
@@ -0,0 +1,37 @@
+# Miden proto build
+
+This crate contains the proto definitions for the gRPC endpoints at `proto/proto`.
+
+Also exposes the compiled proto `FileDescriptorSet` to generate the Rust bindings. See next how to use it.
```

Comment:
I'd change it to something like:

--------------

This crate contains Protobuf files defining the Miden node gRPC API. The files are exposed via `FileDescriptorSet`s to simplify generation of Rust bindings.

It also contains the raw Protobuf files in the `proto` directory to be used for binding generation in other languages.